lib/utility.js

```diff
@@ -66,28 +66,18 @@ export async function loadData(id) {
 	}
 }
 
-export function addWatchPaths(obj) {
-	const paths = globalThis.VITEPRESS_CONFIG.userConfig.themeConfig.dovecot?.watch_paths
+export function dovecotSetting(name) {
+	const setting = globalThis.VITEPRESS_CONFIG.userConfig.themeConfig.dovecot?.[name]
 
-	if (!paths) {
-		throw new Error("Missing 'watch_paths' setting")
+	if (setting === undefined) {
+		throw new Error("Missing '" + name + "' setting")
```

Comment:
This could benefit from template strings:
```js
throw new Error(`Missing '${name}' setting`)
```

Reply:
I personally don't like that syntax.  I would rather use the base string concat formatting, since it's easier to understand.